src/Sentry.Extensions.AI/SentryChatClient.cs

```diff
@@ -5,19 +5,20 @@ namespace Sentry.Extensions.AI;
 internal sealed class SentryChatClient : DelegatingChatClient
 {
     private readonly ActivitySource _activitySource;
-    private readonly HubAdapter _hub = HubAdapter.Instance;
+    private readonly IHub _hub;
```

Comment:
note: make Hub injectable, so that the Tests don't need to rely on the static `SentrySdk`/`HubAdapter`, in order to have the Tests isolated ... so that they could both run in parallel and are independent of the run-order of other tests ... while still using `HubAdapter.Instance` when in production